rdt-app/src/main/java/io/ona/rdt_app/util/RDTJsonFormUtils.java

```diff
@@ -65,7 +65,7 @@ protected ProfileImage doInBackground(Void... voids) {
                 OutputStream os = null;
                 try {
                     if (!StringUtils.isBlank(entityId)) {
-                        final String absoluteFileName = DrishtiApplication.getAppDir() + File.separator + entityId + ".JPEG";
+                        final String absoluteFileName = DrishtiApplication.getAppDir() + File.separator + entityId + "-" + UUID.randomUUID() + ".JPEG";
```

Comment:
With not produce very long paths why not generate UUID and use both as imageId and  path
e.g 

`  final String absoluteFileName = DrishtiApplication.getAppDir() + File.separator +imageIdUUID + ".JPEG";`

Alternatively use directories  

` final String absoluteFileName = DrishtiApplication.getAppDir() + File.separator + entityId + File.separator + UUID.randomUUID() + ".JPEG";`

Reply:
Done.